docs/documentation/stories/include-font-awesome.md

```diff
@@ -15,22 +17,45 @@ Install the `font-awesome` library and add the dependency to package.json...
 npm install --save font-awesome
 ```
 
-### Using CSS
-
 To add Font Awesome CSS icons to your app...
 ```json
 // in angular.json
 "build": {
   "options": {
     "styles": [
-      "node_modules/font-awesome/css/font-awesome.css"
+      "node_modules/font-awesome/css/font-awesome.css",
       "styles.css"
     ],
   }
 }
 ```
+
 ### Using SASS
 
+Create a new project and navigate into the project...
+```
+ng new my-app --style=scss
+cd my-app
+```
+
+Install the `font-awesome` library and add the dependency to package.json...
+```bash
+npm install --save font-awesome
+```
+
+To add Font Awesome CSS icons to your app...
+```json
+// in angular.json
+"build": {
+  "options": {
+    "styles": [
+      "node_modules/font-awesome/css/font-awesome.css",
+      "styles.scss"
+    ],
+  }
+}
+```
+
```

Comment:
There's really two options for SASS.  The most common and probably recommended is the above.  If there is a need to modify any of the font-awesome sass variables (are there any besides the path?) then the second form below (or a variation) would be used.

Could you section off the two and add a note about the second only being useful in special/advanced cases or similar language?

Reply:
The `_variables.scss` and `import` in the section below is not necessary if using the section you added.  They should really be separated out.  The first that you added is generally the preferred and cleaner unless more fine-grained control is needed with the font-awesome [variables](https://github.com/FortAwesome/Font-Awesome/blob/master/web-fonts-with-css/scss/_variables.scss).

Reply:
@clydin I think I am following, I made the minor text separation update. 

I apologize but this is my first contribution. I pushed my updates to the fork on my account, and followed the instruction about rebasing I found here: https://github.com/angular/angular/blob/master/CONTRIBUTING.md
 
However, I am not seeing any commit updates on the Angular account. Is there something else I need to do?

Reply:
Thanks.  Sorry for the delay in response.  The changes look good.  Can you rebase and we can get this in.

Reply:
Awesome! 

@clydin I have never rebased before so I followed contributing.md and ran:
```
git rebase master -i
```
Which returned `Successfully rebased and updated refs/heads/docs-10861.`

Then I ran:
```
git push -f
```
Which returned `Everything up-to-date`

But I am not seeing any sort of changes to my code base or PR. Is there another step?

Reply:
1. Ensure you have an upstream remote configured: https://help.github.com/articles/configuring-a-remote-for-a-fork/
2. Sync your master with upstream: https://help.github.com/articles/syncing-a-fork/
3. Then rebase your branch with your master

Reply:
@clydin I am not sure if I was successful or did something wrong. Should I be seeing every commit since my fork like below?

Reply:
no, you should be seeing them.  I'm not really sure what happened.  Sorry about this but I think it might be best to start over, if possible.